sql/resource_sql_alerts.go

```diff
@@ -46,14 +47,18 @@ func (a *AlertEntity) toCreateAlertApiObject(s map[string]*schema.Schema, data *
 		Op:            a.Options.Op,
 		Value:         a.Options.Value,
 	}
-
-	return ca, nil
+	// This is a workaround for Go SDK problem, will be fixed there.
```

Comment:
Which problem is this?

Reply:
The problem with SDK that it doesn't accept empty string for optional enums

Reply:
Tracked here: https://github.com/databricks/databricks-sdk-go/issues/630